src/rtc/rtc_s1_single_job.py

```diff
@@ -1190,8 +1209,8 @@ def run_single_job(cfg: RunConfig):
         else:
             for pol in pol_list:
                 geo_burst_pol_filename = (f'NETCDF:{output_hdf5_file_burst}:'
-                                          '/science/SENTINEL1/RTC/grids/'
-                                          f'frequencyA/{pol}')
+                                          '/data/'
```

Comment:
What do you think about importing `DATA_BASE_GROUP` from `h5_prep`, and use it instead of `/data`?